src/main/java/com/triangledeterminer/error/ErrorHandler.java

```diff
@@ -0,0 +1,23 @@
+package com.triangledeterminer.error;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+
+import com.triangledeterminer.model.APIError;
+
+@ControllerAdvice
```

Comment:
Do we need this annotiation?

Reply:
Not in this case but if we define Exception Handling globally then we require @ControllerAdvice